src/main/webapp/app/exercises/quiz/manage/drag-and-drop-question/drag-and-drop-question-edit.component.ts

```diff
@@ -761,7 +761,7 @@ export class DragAndDropQuestionEditComponent implements OnInit, OnChanges, Quiz
      * (question text, explanation, hint)
      * @param domainCommands - containing markdownText with the corresponding domainCommand {DomainCommand} identifier
      */
-    domainCommandsFound(domainCommands: [string, DomainCommand][]): void {
+    domainCommandsFound(domainCommands: [string, DomainCommand | null][]): void {
```

Comment:
I'm not sure but probably we should avoid using null. Does it make sense to use undefined here instead, or maybe there's another way to do this?

Reply:
I checked the code and this is correct because it's used by an event emitter that also emits null

Reply:
We checked the code and in this case it makes sense. :D

Reply:
can we change the event emitter to emit `undefined` in a follow-up PR?